pkg/stanza/operator/parser/container/config.go

```diff
@@ -78,6 +88,14 @@ func (c Config) Build(set component.TelemetrySettings) (operator.Operator, error
 		}
 	}
 
+	if removeOriginalTimeField.IsEnabled() {
```

Comment:
Shouldn't we log this message when not using the feature gate instead? That way would make the user aware of the future removal.

```suggestion
	if !removeOriginalTimeField.IsEnabled() {
```

Reply:
Good catch, that was the intention since the gate will be initially in `StageAlpha`.
Thank's for catching this!